modules/manage/pages/console/schema-registry.adoc

```diff
@@ -15,14 +22,156 @@ kafka:
+}
+----
+
+. After registering a schema, you can add a new version to it, delete its subject, or change its compatibility.
+
+=== Topic strategy use case
+
+The Topic strategy is suitable when you want to group schemas by the topics to which they are associated. Suppose you're tracking product order information in a topic named Transactions. When a producer sends records to the OrderInfo topic, you want the record names to look something like: 
+
+- Transactions - Record1 
+- Transactions - Record2 
+
+Where `Record1` and `Record2` are unique identifiers. This is usually defined in your producer settings. Create your schema with the Topic strategy, and the subject name is always Transactions, with all customer transactions under the same topic. 
+
+=== Record strategy use case
+
+The Record strategy is most useful when you have multiple schemas within a topic and need more granular categorization that's influenced by the record name. Suppose there's an Events topic with event types A and B. You may want each of those event types to have their own subject, their own schemas, and their own fully-qualified record names (for example, com.example.EventTypeA). If each event type has its own schema with the Record strategy, then when producers send these event types to the Events topic, their subjects are those record names: 
+
+- com.example.EventTypeA
+- com.example.EventTypeB
+
+The record names in the Events topic looks like this: 
+
+- Events-com.example.EventTypeA-Record1
+- Events-com.example.EventTypeB-Record1
+- Events-com.example.EventTypeA-Record2
+- Events-com.example.EventTypeB-Record2
+
+=== TopicRecord strategy use case
+
+The TopicRecord strategy is suitable when you want to organize schemas based on both topics and logical record types. Suppose there's a microservices architecture where different services produce to the same topic: SharedEvents. Each microservice has a schema of its own for the shared events, but each schema uses the TopicRecord strategy. This results in the following: 
+
+Subject names: 
+
+- SharedEvents-com.example.MicroserviceAEvent
+- SharedEvents-com.example.MicroserviceBEvent
+
+Record names: 
+
+- SharedEvents-com.example.MicroserviceAEvent-Record1
+- SharedEvents-com.example.MicroserviceBEvent-Record1
+- SharedEvents-com.example.MicroserviceAEvent-Record2
+- SharedEvents-com.example.MicroserviceBEvent-Record2
+
+This allows for multiple schemas to govern the same shared events for different microservices, allowing granular organization. 
+
+== Edit compatibility 
+
+include::partial$schema-compatibility.adoc[]
+
+== Delete a schema
+
+Schemas cannot be permanently deleted if any other schemas reference it. They remain readable with a soft delete until you remove all references to it. A soft-deleted schema can be recovered, but a permanently-deleted schema cannot be recovered. Redpanda does not recommend permanently deleting schemas in a production environment.
+
+== End-to-end schema example
```

Comment:
Please extend the context of the schema example.  I'm suggesting a few (intentionally) naive questions to answer in that context:

- Why has the schema a ID =1 and a Topic name Strategy? Is this a context was "staged" in advance to simplify the e2e example? How is it related to the schema management UI tool?
- The producer is a Go code? Which client is it using? Is it franz-go? Is it applicable to producers using another clients/languages? Does the client use the schema registry? The KafkaProducer is configured using bootstrap_servers but there is no sense of schema registry endpoint configuration
- "you would want the compatibility setting to accommodate adding new fields without breakage. Adding a new field to a schema is inherently backward-compatible." Could you explain why?

Reply:
Because I wrote this, I'll answer the question: 

> Why has the schema a ID =1 and a Topic name Strategy? Is this a context was "staged" in advance to simplify the e2e example?

It's "staged" for the purposes of the example. If I can extend it to a particular scenario, let me know. I wanted to simplify it as much as possible so people could see an end-to-end example, but if a different one could be used, I'm happy to. It's not related to the UI yet as I wanted this to get across schema concepts, not how to use them in the UI specifically. 

> The producer is a Go code? Which client is it using? Is it franz-go? Is it applicable to producers using another clients/languages? Does the client use the schema registry? The KafkaProducer is configured using bootstrap_servers but there is no sense of schema registry endpoint configuration

Is this important? Not challenging, just curious. If these contexts should be provided, what would fit best for this example? The assumption is there that the client is using the schema registry, but if more should be added to that code, I'm happy to— I just need more information. 

To clarify, if this is a semantic problem and it shouldn't be called an 'end-to-end' example but something else, happy to do that too.

> "you would want the compatibility setting to accommodate adding new fields without breakage. Adding a new field to a schema is inherently backward-compatible."

This is the case because new consumers will be able to process events written with the new schema, and older consumers will just be able to ignore it— as far as I've read.

@micheleRP  we should add this and also clarify that adding an _optional_ new field to a schema is inherently backward-compatible.